model/task_lifecycle.go

```diff
@@ -849,37 +849,45 @@ func updateMakespans(b *build.Build, buildTasks []task.Task) error {
 	return errors.WithStack(b.UpdateMakespans(depPath.TotalTime, CalculateActualMakespan(buildTasks)))
 }
 
-func getBuildStatus(buildTasks []task.Task) string {
-	// not started
+// getBuildStatus returns a string denoting the status of the build and
+// a boolean denoting if all tasks in the build are blocked.
+func getBuildStatus(buildTasks []task.Task) (string, bool) {
+	// Check if no tasks have started in the build.
```

Comment:
should modify comment to also mention blocked

Reply:
done